app/models/site.rb

```diff
@@ -73,24 +73,35 @@ def self.all_categories_cache
         categories =
           begin
             query =
-              Category
-                .includes(
-                  :uploaded_logo,
-                  :uploaded_logo_dark,
-                  :uploaded_background,
-                  :uploaded_background_dark,
-                  :tags,
-                  :tag_groups,
-                  :form_templates,
-                  category_required_tag_groups: :tag_group,
+              Category.includes(
```

Comment:
We might be approaching a point where we want to extract this and test it in isolation. WDYT?